app/script/calling/CallingRepository.js

```diff
@@ -1604,8 +1604,10 @@ z.calling.CallingRepository = class CallingRepository {
         const expirationDate = new Date(Date.now() + timeout);
         callingConfig.expiration = expirationDate;
 
-        const logMessage = `Updated calling configuration expires on '${expirationDate.toISOString()}'`;
-        this.callLogger.info(logMessage, callingConfig);
+        const turnServersConfig = (callingConfig.ice_servers || []).map(server => server.urls.join('\n')).join('\n');
```

Comment:
Shouldn't we rather join the server array with `, `? That way the log won't be so large.

Reply:
It makes the log way easier to read though, you can easily identify which are tpc and udp servers with the `\n`